source/blog/2018-07-10-Skydive-With-oVirt.html.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Skydive With oVirt 
+author: mburman
+tags: oVirt, oVirt 4.2, open source virtualization,network
+date: 2018-07-10 09:01:00 UTC
+---
+
+## Summary
+
+Skydive is an open source real-time network topology and protocols analyzer providing a comprehensive way of understanding what is happening in your network infrastructure.
+The common use cases will be, troubleshooting, monitoring, SDN integration and much more.
+It has features such as:
+* Topology capturing - Captures network topology, interface, bridge and more
+* Flow capture - Distributed probe, L2-L4 classifier, GRE, VXLAN, GENEVE, MPLS/GRE, MPLS/UDP tunnelling support
+* Extendable - Support for external SDN Controllers or container based infrastructure, OpenStack, OpenContrail, Docker. Supports extensions through API
+* Distributed
+
+[Skydive network](http://skydive.network/) - Link to Skydive site
+
+## Installation steps
+1. git clone https://github.com/skydive-project/skydive.git
+2. Create inventory file
+
+```ini  
+[skydive:children]
+analyzers
+agents
+
+[skydive:vars]
+skydive_listen_ip=0.0.0.0
+skydive_fabric_default_interface=ovirtmgmt
+
+skydive_os_auth_url=https://<FQDN>:35357/v2.0
+skydive_os_service_username=<username>
+skydive_os_service_password=<password>
+skydive_os_service_tenant_name=service
+skydive_os_service_domain_name=Default
+skydive_os_service_region_name=RegionOne
+
+[analyzers]
+<FQDN> ansible_ssh_user=root ansible_ssh_pass=<password>
+
+[agents]
+<FQDN> ansible_ssh_user=root ansible_ssh_pass=<password>
+<FQDN> ansible_ssh_user=root ansible_ssh_pass=<password>
+<FQDN> ansible_ssh_user=root ansible_ssh_pass=<password>
+<FQDN> ansible_ssh_user=root ansible_ssh_pass=<password>
+
+[agents:vars]
+skydive_extra_config={'agent.topology.probes': ['ovsdb', 'neutron'], 'agent.topology.neutron.ssl_insecure': true}
+```
+* skydive_os_auth_url - This is the FQDN(hostname or IP) address of ovirt-provider-ovn
+* skydive_os_service_username - oVirt username used to authenticate the ovirt-provider-ovn, e.g. admin@internal
+* analyzer FQDN will be the hostname of your analyzer
+* agent FQDN will be the hostname of the hosts running in oVirt
+
+3. cd to git/skydive/contrib/ansible
+4. Run ansible-playbook -i <inventory file> playbook.yml.sample
+5. Connect to skydive UI http://<IP/FQDN>:8082
+6. If the UI cannot be accessed flush the iptables 
+  
+## Benefit to oVirt users
+oVirt users may now analyze and see thier network topology via the Skydive UI. They may capture traffic from VM1 to VM2 for example or monitor the traffic between two VMs or hosts. 
+Also they may generate some traffic between 2 running VMs on different hosts for example.
+Users may create alerts in Skydive UI to notify that some traffic disconnected or down.
+
+## Examples
+
```

Comment:
If we would explain two scenarios at the top, they could be used here as an example with screenshots and a short explanation of their presentation in skydive.

Reply:
Agree,  i will do it once we will resolve our issues with neutron networks IDs